packages/cdk8s-cli/lib/import/base.ts

```diff
@@ -44,7 +44,7 @@ export abstract class ImportBase {
 
     for (const name of this.moduleNames) {
       // output the name of the imported resource
-      console.error(name);
+      console.log(name);
```

Comment:
Is that intentional?

Reply:
Yeah, I thought the `error` was unintentional. Any reason for `error` over `log` here?

Reply:
It depends whether we want people to be able to capture stdout and do stuff with it. I actually agree that this specific output should go to stdout so I am okay with this change

Reply:
Sounds good 👍